RuntimeUnityEditor/Windows/ObjectTree/RootGameObjectSearcher.cs

```diff
@@ -119,9 +123,6 @@ public static bool SearchInComponent(string searchString, Component c, bool sear
         {
             if (c == null) return false;
 
-            if (c.ToString().Contains(searchString, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
This check was redundant, the GO name is already searched in the Search function, and it's not needed for component search. Speeds up the search.